common/thorhelper/thorsoapcall.cpp

```diff
@@ -496,7 +496,6 @@ class BlackLister : public CInterface, implements IThreadFactory
                         E->Release();
                         if (stopped.wait(delay))
                             return;
-                        delay += delay;
```

Comment:
This change is also in PR https://github.com/hpcc-systems/HPCC-Platform/pull/17306
But it really needs to be fixed so I added it here also in case that PR doesn't get merged into 8.12.x quickly.

Reply:
I also think delay should be 500 and not 5000, but the other PR lets one change the delay.